tools/cldr-apps/js/src/views/GeneralInfo.vue

```diff
@@ -52,8 +53,23 @@ export default {
     }
 
     this.specialGeneral = cldrText.get("generalSpecialGuidance");
+
+    this.showDashboardOrButton();
   },
+
   methods: {
+    showDashboardOrButton() {
+      // If Dashboard is already visible, hide the button for opening it.
+      if (cldrDashContext.isVisible()) {
+        const els = document.getElementsByClassName("general-open-dash");
+        for (let i = 0; i < els.length; i++) {
+          els[i].style.display = "none";
+        }
```

Comment:
why this and not a reactive vue property?

Reply:
It was based on similar code in cldrDashContext.mjs. I can change it to a property...

Reply:
Changed to a property